lib/players/human.rb

```diff
@@ -0,0 +1,20 @@
+
+
+class Players
+
+    class Human < Player
+
+        def move(board, win = nil)
+            puts "Pick a spot"
+            input = gets
+            if input.match(/^(\d)+$/)
```

Comment:
This method only needs to get a user input. Testing whether the move is valid can be performed in Board#valid_move?